test/src/unit-backwards_compat.cc

```diff
@@ -194,8 +194,22 @@ TEST_CASE(
         array = new Array(ctx, object.uri(), TILEDB_READ);
       }
 
-      // Skip domain types that are unsupported with zipped coordinates.
+      // Skip arrays with heterogeneous dimension types.
       Domain domain = array->schema().domain();
+      REQUIRE(domain.ndim() > 0);
```

Comment:
Are all arrays multidimensional in the unit tests arrays?

Reply:
Yeah -- all arrays are 2D. Both for heterogeneous and homogenous dimensions. This is a sanity check that an array has at least one dimension because otherwise this calculation would segfault.

Reply:
Thanks for clarifying!